shared_model/backend/protobuf/impl/proto_query_response_factory.cpp

```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "backend/protobuf/proto_query_response_factory.hpp"
+#include "backend/protobuf/permissions.hpp"
+#include "backend/protobuf/query_responses/proto_block_query_response.hpp"
+#include "backend/protobuf/query_responses/proto_query_response.hpp"
+
+std::unique_ptr<shared_model::interface::QueryResponse>
+shared_model::proto::ProtoQueryResponseFactory::createAccountAssetResponse(
+    std::vector<std::shared_ptr<shared_model::interface::AccountAsset>> assets,
+    const crypto::Hash &query_hash) {
+  iroha::protocol::QueryResponse protocol_query_response;
+  protocol_query_response.set_query_hash(query_hash.hex());
```

Comment:
Please create a common method for this, so these lines are not duplicated in all the methods.
```C++
iroha::protocol::QueryResponse protocol_query_response;
protocol_query_response.set_query_hash(query_hash.hex());
...
return std::make_unique<shared_model::proto::QueryResponse>(
      std::move(protocol_query_response));
```
Maybe create a method, so that it is possible to pass something in place of `...`?

Reply:
Imo it's better leave as is, otherwise code will be harder to read and modify